### DISCUSSION TITLE
Could we add vocabulary for used terms?

### DISCUSSION THREAD

**Discussion by ezr-ondrej** [foresterorg/forester#2] [General]
I think applience for example does not have a clear meaning to everyone, we could also use it to explain more technical terms like out-of-bound etc :)

**Comment by lzap** [comment]
Absolutely, btw it is out-of-band :-)

Let’s first start discussion about finding the right terms because I am struggling with many, then we can add this to wiki or something.

- appliance
- image
- system
- acquire
- release

Hmm anything else? I am biases I live in this environment for quite some time.
